pastas/recharge.py

```diff
@@ -345,14 +354,14 @@ def get_root_zone_balance(pe, ep, srmax=250.0, lp=0.25, ks=100.0,
         """
         n = pe.size
         # Create empty arrays to store the fluxes and states
-        sr = zeros(n + 1, dtype=float64)  # Root Zone Storage State

```

Comment:
This needs to be updated. I think the changes for this PR were made on an old version of the dev-branch. This change is unnecessary and will cause water balance errors at the final time step.

Reply:
This is weird. On my local (up to date) unsat branch I have the newest version of the Flex Model. But in this PR it shows as if I overwrite these changes.

Reply:
Nevermind. I'll fix it.